packages/pigment-css-nextjs-plugin/src/index.ts

```diff
@@ -61,13 +61,7 @@ export function withPigment(nextConfig: NextConfig, pigmentConfig?: PigmentOptio
           if (what.startsWith('__barrel_optimize__')) {
             return require.resolve('../next-font');
           }
-          // Need to point to the react from node_modules during eval time.
-          // Otherwise, next makes it point to its own version of react that
-          // has a lot of RSC specific logic which is not actually needed.
-          if (what.startsWith('@babel') || what.startsWith('react') || what.startsWith('next')) {
-            return require.resolve(what);
-          }
-          if (what === 'next/image') {
+          if (what === 'next/image' || what === 'next/link') {
```

Comment:
If `what==='next/link'` we still import `next-image`? @brijeshb42

Reply:
Both just point to a dummy module for wyw evaluation to work.

Reply:
next/link just provides a default export which is already there in the next-image file.

Reply:
OK :ok_hand:

Reply:
That's a bit confusing 🙂

Reply:
We could just rename the next-image file to `next-module-stub` or similar to remove the confusion.
It's on our side and not user facing.